docs/current/getting-started/index.md

```diff
@@ -4,9 +4,15 @@ lang: en
 title: Getting Started
 ---
 
-## Prerequisites
+## Fast Installation
 
-To develop in the Substrate ecosystem, you must set up your developer environment. Depending on your operating system, these instructions may be different.
+Install all the required dependencies with a single command. (Be patient, this can take up to 30 minutes)
```

Comment:
@JoshOrndorff actually this has been long discussed in many forums. i suggest you speak with joe or ben about this, but yes, this absolutely takes a very long time (more than 30 minutes actually). this is why its been removed from the home page, and will be replaced with just a link to the playground...

Reply:
Discussions and appeals to authority are great in some cases, but in this one we can just run the command and see. Here's a video of me running it on a fresh Ubuntu VPS (8gb, 4 core). It took 3 minutes.

https://www.youtube.com/watch?v=tjZQP1VLwzI

Like our discussion about whether the "prerequisites" are included in this script, it's better to find out for ones self.

Reply:
@JoshOrndorff im glad it went fast for you in your video. I still suggest you speak with @joepetrowski about this because I did try it twice (on macs) and it did take a very long time. Unfortunately,  I'm not the one you need to convince here.

Reply:
Definitely a Mac problem, also takes about 1 minute on my Manjaro laptop.

Reply:
ah its primarily a mac problem, thank you for clearing that up @joepetrowski 
Would you guys prefer to explicitly state that? (i.e., `For Mac Users: Be patient, this can take up to 30 minutes`

Reply:
Yeah, that works for me